php/editor.php

```diff
@@ -32,11 +32,15 @@ function enqueue_code_editor( $type, $extra_atts = [] ) {
 	$default_atts = [
 		'mode'          => $modes[ $type ],
 		'matchBrackets' => true,
-		'extraKeys'     => [ 'Alt-F' => 'findPersistent', 'Ctrl-Space' => 'autocomplete' ],
-		'gutters'       => [ 'CodeMirror-lint-markers' ],
+		'extraKeys'     => [ 
+							'Alt-F' => 'findPersistent',
+							'Ctrl-Space' => 'autocomplete',
+						],
+		'gutters'       => [ 'CodeMirror-lint-markers', 'CodeMirror-foldgutter'],
 		'lint'          => 'css' === $type || 'php' === $type,
 		'direction'     => 'ltr',
 		'colorpicker'   => [ 'mode' => 'edit' ],
+		'foldOptions' 	=> [ 'widget' => '...' ],
```

Comment:
The default widget looks really out of place. This looks much cleaner. The only caveat is that it must be manually added in the preview editor over at the settings page as well.